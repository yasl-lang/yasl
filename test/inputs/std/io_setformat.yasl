const setformat = io.setformat

const l = [ 10, '20', 30, '40']

echo l

setformat('r')
echo l

setformat('x')
echo [ 10, 20, 30 ]

setformat()
echo l
