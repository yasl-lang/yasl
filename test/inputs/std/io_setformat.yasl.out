[10, 20, 30, 40]
[10, '20', 30, '40']
[0xa, 0x14, 0x1e]
[10, 20, 30, 40]
